SU-SWS/decanter-react
# ADAPT-2102: Hero component

# READY FOR REVIEW

# Summary
- Add Hero component

# Needed By (Date)
- tbd

# Urgency
- low

# Steps to Test

1. Checkout this branch
2. Spin up local dev to review the Hero or see on [Netlify Preview](https://deploy-preview-56--decanter-react.netlify.app/?path=/story/composite-hero--default)

# Associated Issues and/or People
- [ADAPT-2102](https://stanfordits.atlassian.net/browse/ADAPT-2102)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
